.github/workflows/ci-cd.yaml

```
@@ -77,6 +81,7 @@ jobs:
   deploy-staging:
     name: Internal staging
     runs-on: ubuntu-latest
+    environment: staging
     if: always() && !failure() && !cancelled() 
```

yamllint

[error] trailing spaces

(trailing-spaces)